styles.css

```diff
@@ -458,3 +460,399 @@ form {
+    font-size: 0.8vw;
+    font-weight: 50;
+    padding: 0.07rem 0.3rem 0.07rem 0.3rem;
+    border-radius: 0.3rem;
+  }
+
+  .buttons > ul {
+    margin: 0;
+  }
+
+  .container > ul > .name {
+    font-size: 1.1vw;
+  }
+
+  .dropdown-content > a > img {
+    padding: 0;
+    width: 4.5vw;
+    grid-row: 1/2;
+    grid-column: 1/2;
+  }
+
+  .socials > ul > li > a > img {
+    width: 1.5vw;
+  }
+
+  /* about myself */
+  #headline-two {
+    display: grid;
+    grid-template-columns: repeat(2, 40% 60%);
+    grid-template-rows: repeat(9, 5vh);
+    height: 55vh;
+    padding: 8.2% 8.2% 4.2% 8.2%;
+    background-color: #fff;
+    border-radius: 0 20% 0 0;
+  }
+
+  .wrapper-contact > div > h2 {
+    font-size: 3vw;
+  }
+
+  #headline-two > h2 {
+    grid-row: 8/9;
+  }
+
+  #headline-two > .socials {
+    grid-row: 9/10;
+    align-self: center;
+  }
+
+  #about-myself {
+    height: 60vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-image: none;
+    background-color: #f7f7f9;
+  }
+
+  #resume {
+    grid-row: 10/11;
+    width: 13vw;
+    padding: 0;
+  }
+
+  .dropbtn {
+    font-size: 1.4vw;
+    padding: 1rem;
+  }
+
+  .dropdowns {
+    grid-column: 2 / 3;
+    grid-row: 2/6;
+  }
+
+  .list-of-dropdowns {
+    margin: 0;
+  }
+
+  .dropdown-content {
+    display: none;
+  }
+
+  .list-of-dropdowns > li:hover .dropdown-content {
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+  }
+
+  .dropdown-content > a {
+    font-size: 1.3vw;
+    height: 7.5vw;
+    width: 7.5vw;
+    display: grid;
+    grid-template-rows: 70% 30%;
+    padding: 2%;
+  }
+
+  .dropdown-content > a > span {
+    grid-row: 2/3;
+    grid-column: 1/3;
+    font-weight: 250;
+    color: #253858;
+  }
+
+  /* contact form */
+
+  #contact-form {
+    height: 65vh;
+    margin: 0;
+    padding: 0;
+    background-color: #fff;
+    margin-top: 10%;
+  }
+
+  .wrapper-contact {
+    background-image: url(images/desktop-pictures/contact\ form\ background\ shapes.png);
+    background-size: contain;
+    border-radius: 20% 0 0 0;
+    padding: 3%;
+    margin: 0;
+    height: 80vh;
+  }
+
+  .wrapper-contact > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 55vw;
+    height: 70vh;
+    justify-self: flex-start;
+    padding-top: 6%;
+  }
+
+  .contact-para {
+    width: 30vw;
+    font-size: 1.5vw;
+    padding: 0;
+    padding-bottom: 2%;
+    line-height: inherit;
+  }
+
+  #contact-form > .wrapper-contact {
+    height: 60vh;
+    margin-top: 0;
+    align-items: center;
+  }
+
+  .wrapper-contact > * {
+    font-size: 1rem;
+    width: 70vw;
+    padding: 0;
+    justify-self: start;
+  }
+
+  form {
+    padding: 0;
+    margin: 0;
+    width: 45vw;
+    height: 30vh;
+  }
+
+  input[type=text],
+  input[type=email] {
+    width: 22vw;
+    height: 1vh;
+    font-size: 1.5vw;
+    border-radius: 2px;
+  }
```

Comment:
- [ ] Please make sure to add some padding to your input fields in the form as they don't look similar to the Figma design. 🎨
- Your design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/68afa7cd-f9ee-44ec-92ff-c7c4dc5feea0)
- Figma design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/737b800d-a854-4bd6-9be1-24713d228264)